### PR TITLE
CLDR-16945 BRS v44: re-import of data from ST

### PR DESCRIPTION
**Pull request by srl295**
- skipped: aa, mt (would reintroduce errs)

CLDR-16945

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/main/km.xml [hunk 1]
```diff
@@ -4640,17 +4640,6 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<symbols numberSystem="khmr">
 			<decimal>,</decimal>
 			<group>.</group>
-			<list draft="contributed">↑↑↑</list>
```
Comment (AEApple) [inline common/main/km.xml:4643]: So I didn't vote for explicit inheritance here. Should we revert the km import, or should I go in and vote for explicit inheritance?

Comment (srl295) [reply common/main/km.xml:4643]: That i'm not sure, but i think for now we should just fix it in XML if needed?

Comment (AEApple) [reply common/main/km.xml:4643]: Wouldn't it just automatically inherit the default numbering system formatting even if it's not explicitly present?

Comment (srl295) [reply common/main/km.xml:4643]: i'd think so. it'll just be missing next time.

**Comment by srl295** [comment]
@AEApple  OK to merge this?

**Review by AEApple — APPROVED** [review]

**Comment by AEApple** [comment]
Yeah, we can always fix km in xml later.

**Pull request merged**